graphql_compiler/tests/integration_tests/test_backends_integration.py

```diff
@@ -26,12 +29,36 @@
     test_backend.MYSQL,
     test_backend.SQLITE,
     test_backend.MSSQL,
+    test_backend.NEO4J,
+    test_backend.REDISGRAPH,
+])
+
+# TODO Leon this is really bad but it works for now
+all_backends_but_redisgraph = parameterized.expand([
+    test_backend.ORIENTDB,
+    test_backend.POSTGRES,
+    test_backend.MARIADB,
+    test_backend.MYSQL,
+    test_backend.SQLITE,
+    test_backend.MSSQL,
+    test_backend.NEO4J,
+])
+
+all_backends_but_cypher = parameterized.expand([
```

Comment:
I am not a big fan of this idea. I prefer having a 
if backend==test_backend.NEO4J:
    return

Reply:
See commit aedc4a9 for fix